docs/pages/docs/guides/migrate-from-lerna.mdx

```diff
@@ -1,3 +1,8 @@
+---
+title: Migrate from Lerna
+description: Turborepo and Lerna have several features that are common to both. However, they are different in many ways as well. This quick and handy tutorial explains how to use Turborepo with Lerna.
```

Comment:
```suggestion
description: Learn how to incrementally adopt Turborepo from your existing Lerna monorepo.
```

Feedback: If we say "However, they are different in many ways as well" - it's better to just explain _what_ those differences are, or don't mention lines like that 👍